### ISSUE TITLE
Removed quoation marks from query.

### ISSUE THREAD

**Issue by ckissi** [jarektkaczyk/eloquence#119]
Hello,

I tried to use it one one of my models unfortunately it doesn't use quoation mark inside the query. Here is the example code from error page I got:

`SQLSTATE[42000]: Syntax error or access violation: 1055 'developer_dev.dilemmas.user_id' isn't in GROUP BY (SQL: select * from (select `dilemmas`.*, max(case when `dilemmas`.`title` = audi or `dilemmas`.`title` = bmw then 15 else 0 end + case when `dilemmas`.`title` like audi% or `dilemmas`.`title` like bmw% then 5 else 0 end + case when `dilemmas`.`title` like %audi% or `dilemmas`.`title` like %bmw% then 1 else 0 end) as relevance from `dilemmas` where (`dilemmas`.`title` like %audi% or `dilemmas`.`title` like %bmw%) group by `dilemmas`.`id`) as `dilemmas` where `relevance` >= 0.25 order by `relevance` desc) `

As you can see there are no " around the search terms  'audi'

**Comment by jarektkaczyk** [comment]
@ckissi it's not about the `?` (which is used ofc), but about your MySQL settings.

tldr; disable strict mode

Read https://mattstauffer.co/blog/strict-mode-and-other-mysql-customizations-in-laravel-5-2 and http://rpbouman.blogspot.sg/2014/09/mysql-575-group-by-respects-functional.html and http://dev.mysql.com/doc/refman/5.7/en/sql-mode.html

**Comment by ckissi** [comment]
THANK YOU MAN !!! super fast response. Works already :)

**Comment by ganey** [comment]
Just as a note, strict mode was causing me GROUP BY errors on relations.

Working fine since turning it off in the Laravel config.

**Comment by doctorallen** [comment]
Is there any way around this without turning off strict mode?

**Comment by jarektkaczyk** [comment]
Nope

Jarek Tkaczyk

softonsofa :: development with pleasure

**Comment by doctorallen** [comment]
Do you know why it does this? If you know the source of the problem I could try a PR?
